<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BetControls</class>
 <widget class="QWidget" name="BetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_check">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_bet">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Bet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_fold">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Fold</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_autocheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto - Check</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
